addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -742,6 +742,7 @@ def _import_fill_invoice_line_form(self, journal, tree, invoice, invoice_line, q
         if not tax_nodes:
             for elem in tree.findall('.//{*}TaxTotal'):
                 tax_nodes += elem.findall('.//{*}TaxSubtotal/{*}TaxCategory/{*}Percent')
+                tax_nodes += elem.findall('.//{*}TaxSubtotal/{*}Percent')
```

Comment:
you are duplicating the nodes and so, the taxes :grimacing:
```suggestion
                percentage_nodes = elem.findall('.//{*}TaxSubtotal/{*}TaxCategory/{*}Percent')
                if not percentage_nodes:
                    percentage_nodes = elem.findall('.//{*}TaxSubtotal/{*}Percent')
                tax_nodes += percentage_nodes
```